documentation/modules/oauth/con-mapping-keycloak-authz-services-to-kafka-model.adoc

```diff
@@ -0,0 +1,92 @@
+[id="con-mapping-keycloak-authz-services-to-kafka-model_{context}"]
+= Mapping Keycloak Authorization Services to the Kafka authorization model
+
+[role="_abstract"]
+
+To grant Kafka permissions to user accounts or service accounts, use Keycloak Authorization Services rules on the OAuth client that represents the Kafka Broker.
+Typically, the client id of the broker OAuth client is `kafka`.
```

Comment:
In the example provided with Strimzi it is. Perhaps we should say that and that you should make sure the Kafka client is easy it identify.

Reply:
You can use the same OAuth client ('kafka') with all your Kafka clusters. This gives you a single unified space of authorization rules. But you can also use different OAuth client ('my-cluster-kafka', 'cluster-dev-kafka' or something such) and define authorization rules specific to each cluster within each client configuration).

Reply:
Thanks. I summarised this in a note.